src/phoenix/datasets/experiments.py

```diff
@@ -118,6 +119,48 @@ def _phoenix_client() -> httpx.Client:
     return client
 
 
+class JSONParsable(ExperimentEvaluator):
+    annotator_kind = "CODE"
+
+    def __call__(
+        self, input: JSONSerializable, reference: JSONSerializable, output: JSONSerializable
+    ) -> EvaluationProtocol:
+        assert isinstance(output, str), "Experiment run ooutput must be a string"
+        try:
+            json.loads(output)
+            json_parsable = True
+        except json.JSONDecodeError:
+            json_parsable = False
+
+        return EvaluationProtocol(
+            score=float(json_parsable),
+            explanation=None,
+            metadata={},
+        )
+
+
+class ContainsKeyword(ExperimentEvaluator):
+    annotator_kind = "CODE"
+
+    def __init__(self, contains: str):
```

Comment:
```suggestion
    def __init__(self, substring: str):
```

`contains` sounds like a boolean.

Reply:
good point, going with "keyword" to match the object verbiage